XRTK-Core/Packages/com.xrtk.core/Editor/MixedRealityToolkitInspector.cs

```diff
@@ -19,9 +19,14 @@ namespace XRTK.Editor
     [CustomEditor(typeof(MixedRealityToolkit))]
     public class MixedRealityToolkitInspector : UnityEditor.Editor
     {
+        private const string ObjectSelectorClosed = "ObjectSelectorClosed";
```

Comment:
These should be camel case.

Reply:
It's a const

Reply:
Probably should be all caps but it doesn't really matter imho

Reply:
We recently agreed to use camel case for private const. But not too invested here. We should just stick to something I guess.

Reply:
Mainly I just wanted it to match the exact string

Reply:
![Screenshot_20210419-023515_Chrome](https://user-images.githubusercontent.com/13334553/115191868-0730d700-a0b8-11eb-8cbf-a1dc6b9f8fc3.png)
https://stackoverflow.com/questions/242534/c-sharp-naming-convention-for-constants